test/test_files/tinysnb/function/table.test

```diff
@@ -80,3 +80,11 @@ height
 -STATEMENT CALL db_version() RETURN version
 ---- 1
 v0.0.9
+
+-LOG ReturnTableRelation
```

Comment:
test show_relation on node table

Reply:
I think when calling show_relation on node table. Return 0 rows is a better behaviour